examples/fastly/src/index.js

```diff
@@ -0,0 +1,79 @@
+/// <reference types="@fastly/js-compute" />
+import { connect } from '@planetscale/database'
+
+
+addEventListener("fetch", (event) => event.respondWith(app(event)))
+
+fastly.defaultBackend = "psdb"
+const CURRENT_YEAR = 2022
+/**
+ *
+ * @param {FetchEvent} event
+ * @returns {Promise<Response>}
+ */
+async function app(event) {
+  const secrets = new Dictionary('secrets')
```

Comment:
Just a question for my understanding, is this available on the global scope outside this function? If so, the better pattern would be to define:

```javascript
const db = new Client({ ... })
```

globally, then within the event handler, do `const conn = db.connection()`.

Concretely, this is what I was able to do for Vercel: https://github.com/planetscale/f1-championship-stats/blob/main/examples/vercel/api/data.json.ts#L9-L11

Some of the runtimes pass env vars and whatnot through the handler, so it's not really as feasible to have a global pool.

Reply:
This is not constructable outside the handler as the top level scope is executed once during the wasm construction prior to uploading to fastly and so does not have access to the dictionary

Reply:
Gotcha, that makes sense!